lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -336,52 +357,59 @@ def self.available_options
                               verify_block: proc do |value|
                                 accepted_formats = [".apk"]
                                 file_extname_full = Helper::AppcenterHelper.file_extname_full(value)
-                                UI.user_error!("Only \".apk\" formats are allowed, you provided \"#{file_extname_full}\"") unless accepted_formats.include? file_extname_full
+                                self.optional_error("Only \".apk\" formats are allowed, you provided \"#{file_extname_full}\"") unless accepted_formats.include? file_extname_full
                               end),
 
           FastlaneCore::ConfigItem.new(key: :aab,
```

Comment:
I assume we are keeping this now for backward compatibility ?
Whats the plan to remove this ?

Reply:
No plan to remove. Just marked as deprecated. :)

Reply:
To be clear, to remove we would have to change the major version number